static/usage/breadcrumbs/styling/color/vue.md

```diff
@@ -0,0 +1,19 @@
+```html
+<template>
+  <ion-breadcrumbs color="primary">
```

Comment:
The Vue example does not work 
![image](https://user-images.githubusercontent.com/2721089/172655502-4f578e4f-3c63-4928-8bcd-24b3ddb1f55c.png)

Reply:
Is this possibly a bug? I'm seeing the color class being applied to `ion-breadcrumbs`, but `ion-breadcrumb` has it's own host level CSS variable values for the colors. 

Looks that `.in-breadcrumbs-color` is not being applied to the individual `ion-breadcrumb` components (only in the Vue example).

Reply:
Haven't been able to track down the cause; went ahead and made an issue for it: https://github.com/ionic-team/ionic-framework/issues/25446 I'll move the `color` attribute to the inner `ion-breadcrumb` elements as a workaround for now.